examples/00-mapdl-examples/bracket_static.py

```diff
@@ -0,0 +1,743 @@
+#
+# To create the area delineated by ``line1``, ``line2`` and newly created
+# ``line3``, we use the :meth:`mapdl.al() <ansys.mapdl.core.Mapdl.al>` command.
+# The three lines are the input. If we select them all, we can use the ``'ALL'``
+# argument to create the area.
+#
+# First we have to reselect the newly created lines in the fillet area.
+# To do this we can use the fillet_radius parameter and the
+# :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command.
+#
+# For the two newly created straight lines, the length will be the same as the
+# fillet_radius. So we can use the length argument with the
+# :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command.
+#
+
+mapdl.allsel()
+
+# Select lines for the area
+mapdl.lsel("S", "LENGTH", "", fillet_radius)
+
+###############################################################################
+# Additionally, we need the fillet line itself (``line3``) but we can use the
+# :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command again with either
+# the ``'RADIUS'`` argument if there is only one line with that radius in our model
+# or more directly using the parameter name of the line.
+# Note the ``'A'`` to additionally select items.
+#
+mapdl.lsel("A", "LINE", "", line3)
+
+# plotting ares
+mapdl.lplot(vtk=True, cpos="xy", show_line_numbering=True)
+
+###############################################################################
+# Then we use :meth:`mapdl.al() <ansys.mapdl.core.Mapdl.al>` to create the areas
+# from the lines.
+#
+
+# Create the area
+mapdl.al("ALL")  # Prints the id of the newly created area
+
+
+###############################################################################
+# Add areas together
+# ~~~~~~~~~~~~~~~~~~
+# Append all areas again with the :meth:`mapdl.aadd() <ansys.mapdl.core.Mapdl.aadd>`.
+# Since we have only the 2 areas to combine, use the ``'ALL'`` argument.
+#
+
+# Add the area to the main area
+mapdl.aadd("all")
+mapdl.aplot(vtk=True, cpos="xy", show_area_numbering=True, show_lines=True)
+
+###############################################################################
+# Create first pin hole
+# ~~~~~~~~~~~~~~~~~~~~~
+#
+# The first pin hole is located at the left side of the first box, so we can use
+# the box dimensions to locate our new circle.
+#
+# The X value (center) of the pinhole is at the first coordinate of the ``box1``
+# (``X1``). The Y value is the average of the two ``box1`` Y values:
+#
+
+# Create the first pinhole
+pinhole_radius = 0.4
+pinhole1_X = box1[0]
+pinhole1_Y = (box1[2] + box1[3]) / 2
+
+pinhole1 = mapdl.cyl4(pinhole1_X, pinhole1_Y, pinhole_radius)
+
+###############################################################################
+#
+# Since we have two pin hole circles, we will use the command twice.
+#
+# **Note** some of these areas are set to parameters to use later in the analysis.
+# This will later allow us to these lines that are used to create the areas
+# :meth:`mapdl.asll() <ansys.mapdl.core.Mapdl.asll>`
```

Comment:
```suggestion
# Because you have two pin hole circles, you use the command twice.
#
# **Note** some of these areas are set to parameters to use later in the analysis.
# This allows you to use the lines to create the areas with the
# :meth:`mapdl.asll() <ansys.mapdl.core.Mapdl.asll>` command.
```